packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.controller.js

```diff
@@ -16,7 +16,7 @@ export default class {
 
   $onInit() {
     this.loading = false;
-    this.nsx_sizes = Object.keys(EDGES_SIZES);
```

Comment:
Is not used anywhere ?

Reply:
It's ok @tibs245 https://github.com/ovh/manager/blob/feat/custom-nsx-edges-sizing-bis/packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.html#L49